templates/chrony.conf.epp

```diff
@@ -0,0 +1,127 @@
+# NTP servers
+<% $chrony::servers.each |$server| { -%>
+server <%= $server.flatten.join(' ') %>
+<% } -%>
```

Comment:
This is not good enough when $chrony::servers is a hash.  It (kind of) works, because
.each() returns a Array ['key','value']. But if I use hieradata with a hash without value,
this will result in a line ending with a space.

Example:

```hiera
chrony::servers:
  ntp1.example.com:
    - minpoll 3
    - maxpoll 7
  ntp2.example.com:
```

It results in server "ntp1.example.com minpoll 3 maxpoll 7", but "server ntp2.example.com "

I'm going to think some about this, and submit a PR with a fix later. We should also have
a test that would have detected this.